src/steps/os/windows.rs

```diff
@@ -77,103 +131,19 @@ pub fn update_wsl(ctx: &ExecutionContext) -> Result<()> {
     print_separator(t!("Update WSL"));
 
     let mut wsl_command = ctx.run_type().execute(wsl);
-    wsl_command.args(["--update"]);
+    wsl_command.args(WSL_UPDATE);
 
     if ctx.config().wsl_update_pre_release() {
-        wsl_command.args(["--pre-release"]);
+        wsl_command.arg("--pre-release");
     }
 
     if ctx.config().wsl_update_use_web_download() {
-        wsl_command.args(["--web-download"]);
+        wsl_command.arg("--web-download");
     }
     wsl_command.status_checked()?;
     Ok(())
 }
 
-/// Detect if WSL is installed or not.
-///
-/// For WSL, we cannot simply check if command `wsl` is installed as on newer
-/// versions of Windows (since windows 10 version 2004), this commmand is
-/// installed by default.
-///
-/// If the command is installed and the user hasn't installed any Linux distros
-/// on it, command `wsl -l` would print a help message and exit with failure, we
-/// use this to check whether WSL is install or not.
```

Comment:
As a non-Windows user, reviewing Windows change is pretty hard for me, please don't remove these explanation comments, without them, maintenance would be seriously harder. And, if possible, please add more such comments to make Topgrade maintainable:)